src/web/app/pages-student/student-profile-page/student-profile-page.component.html

```diff
@@ -3,21 +3,27 @@ <h1>Student Profile</h1>
   <div class="card">
     <div class="card-body fill-plain">
       <h1>{{ name }}</h1>
-      <div class="form-group row">
-        <div class="col-xs-6 col-sm-5 col-md-3 cursor-pointer">
+      <div class="row profile-pic-row">
+        <div class="col-3 cursor-pointer">
           <img
               id="profilePic"
-              src="{{ getProfilePictureUrl(student.studentProfile.pictureKey) }}"
-              class="profile-pic">
+              src="{{ profilePicLink }}"
+              class="profile-pic"
+              (click)="onUploadEdit()">
         </div>
-        <div class="col-xs-6 col-sm-5 col-md-3" ngbTooltip="Upload a profile picture (.jpg or .png, max size 5 MB)">
-          <!-- TODO: Include upload/edit photo functionality
+        <div class="col-2 btn-group-vertical" ngbTooltip="Upload a profile picture (.jpg or .png, max size 5 MB) or delete your profile picture">
           <button
               id="uploadEditPhoto"
               class="btn btn-primary"
-              type="button">
+              (click)="onUploadEdit()">
             Upload/Edit Photo
-          </button> -->
+          </button>
+          <button
+              id="deletePhoto"
+              class="btn btn-danger"
+              (click)="onDelete(confirmDeleteProfilePicture)">
```

Comment:
This button should not be visible if there are no profile picture in the first place.